collective/searchandreplace/searchreplaceutility.py

```diff
@@ -426,7 +436,7 @@ def getRawTextField(obj, field):
         if isinstance(baseunit, tuple):
             #  LinesField
             text = "\n".join(baseunit)
-        elif isinstance(baseunit.raw, unicode):
+        elif hasattr(baseunit, 'raw') and isinstance(baseunit.raw, unicode):
             text = baseunit.raw
         else:
             text = _to_unicode(field.getRaw(obj))
```

Comment:
Related, but not changed in this PR: I don't think the `_to_unicode` function does anything on Python 3. And we may want to use `safe_unicode` from `CMFPlone.utils` instead. But it's optional, just a remark now that I look at the code.

Reply:
Done in separate commit